### PR TITLE
DROOLS-6997 Move printStackTrace() to SLF4J logging

### PR DESCRIPTION
**Pull request by tarilabs**
**JIRA**: https://issues.redhat.com/browse/DROOLS-6997

**referenced Pull Requests**: none

<details>
<summary>
How to replicate CI configuration locally?
</summary>

Build Chain tool does "simple" maven build(s), the builds are just Maven commands, but because the repositories relates and depends on each other and any change in API or class method could affect several of those repositories there is a need to use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
 
[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

File: kie-api/src/main/java/org/kie/api/pmml/PMML4Result.java [hunk 1]
```diff
@@ -192,7 +196,7 @@ public Object getResultValue(String objName, String objField, Object...params) {
                         }
                     }
                 } catch (SecurityException | IllegalAccessException | IllegalArgumentException | InvocationTargetException e1) {
-                    e1.printStackTrace();
+                    LOG.error("Exception", e1);
```
Comment (tarilabs) [inline kie-api/src/main/java/org/kie/api/pmml/PMML4Result.java:199]: to be noticed logger per usual is declared static so shouldn't influence on JAXB marshalling

File: kie-internal/src/main/java/org/kie/internal/security/KiePolicyHelper.java [hunk 2]
```diff
@@ -60,7 +60,6 @@ public class KiePolicyHelper {
         } catch (Exception e) {
             ctx = null;
             log.error("Error loading and installing KIE security policy.", e);
-            e.printStackTrace();
```
Comment (tarilabs) [inline kie-internal/src/main/java/org/kie/internal/security/KiePolicyHelper.java:63]: the exception is already captured in SLF4J logging appropriately in the line above

**Review by mariofusco — APPROVED** [review]

File: kie-api/src/main/java/org/kie/api/pmml/PMML4Result.java (hunk 1 above)
Comment (gitgabrio) [reply kie-api/src/main/java/org/kie/api/pmml/PMML4Result.java:199]: :+1:

**Comment by tarilabs** [comment]
`10:21:17  [ERROR]  You have exceeded a secondary rate limit. Please wait a few minutes before you try again.`

Jenkins run cdb

**Comment by tarilabs** [comment]
`10:21:19  [ERROR]  You have exceeded a secondary rate limit. Please wait a few minutes before you try again.`

Jenkins run fdb

**Comment by tarilabs** [comment]
Jenkins run cdb

**Comment by tarilabs** [comment]
Jenkins run fdb

**Comment by tarilabs** [comment]
image

SnRxGW

**Pull request merged**